ra/ra.go

```diff
@@ -625,10 +644,19 @@ func (ra *RegistrationAuthorityImpl) checkInvalidAuthorizationLimit(ctx context.
 	// Most rate limits have a key for overrides, but there is no meaningful key
 	// here.
 	noKey := ""
-	if count.Count >= limit.GetThreshold(noKey, regID) {
+	threshold, isOverridden := limit.GetThreshold(noKey, regID)
+	if count.Count >= threshold {
 		ra.log.Infof("Rate limit exceeded, InvalidAuthorizationsByRegID, regID: %d", regID)
+		if isOverridden {
+			utilization := float64(count.Count) / float64(threshold)
+			ra.rlOverrideUsageGauge.WithLabelValues(ratelimit.InvalidAuthorizationsPerAccount, strconv.FormatInt(regID, 10)).Set(utilization)
+		}
```

Comment:
Move this (and the identical code from line 656) up to between lines 647 and 648, so it only has to appear once.

Reply:
Oh whoops, this was left in from my first approach. This should have been identical to the other implementations.